README.md

```diff
@@ -1056,6 +1059,29 @@ Default: `false`.
 > * [Reconfigure](http://msdn.microsoft.com/en-us/library/ms176069.aspx)
 > * [Server Configuration Options](http://msdn.microsoft.com/en-us/library/ms189631.aspx)
 
+### Bolt Tasks
+
+#### get_sql_logins
+
+This task will retrieve information about a login, or a set of logins, from the sql instances running on a given node.
+With no parameters specified it will return summary level information about all logins configured for all sql instances running on a given node or node set. Use the `detailed` parameter to return more detailed information including the SID's and the name of the instance a login was retrieved from.
+
+##### parameters
+
+* **instance_name**
+    The name of the instance to query for logins. By default, leave blank for all instances running on a node. Pass the values `.`, `MSSQLSERVER`, or the node name to query just the default instance. Named instances can be referred to by either the short name of the instance, or by `<COMPUTERNAME>\<INSTANCE_NAME>`. This parameter will accept nothing, a single string, or an array of strings.
+
+* **login_name**
+    The name of a particular login to search for, or the search pattern for a set of logins. If no value is passed to this variable then all logins are returned. By default any values passed to this parameter are treated like a search string. Searches are done using the PowerShell `-match` operator. For example, if the string `sql` is passed to this parameter,logins such as `NT SERVICE\SQLWriter` and `##MS_PolicyTsqlExecutionLogin##` will be returned in the result set. If the `exact_match` parameter is set to true, then only exact matches are accepted and neither of those logins would have been returned. This parameter will accept nothing, a single string, or an array of strings.
+
+* **exact_match**
+    Set this to true to change the behavior of the `login_name` parameter so that only logins exactly matching one of the provided login_names is returned in the result set. This is a boolean parameter that accepts only true or false.
```

Comment:
Suggest a rewording of the last sentence to:

> This is an optional parameter which will accept either `true` or `false`.

The default behavior should also be clarified - it's implied to be `false`.